test/lib/github.ts

```diff
@@ -49,5 +49,47 @@ describe('github', () => {
       }
       request.done();
     });
+
+    it('does not return latest release without prefix, when prefix used', async () => {
+      const request = nock('https://api.github.com')
+        .get('/repos/bcoe/test/tags?per_page=100&page=1')
+        .reply(200, [{name: 'v1.0.2'}])
+        .get('/repos/bcoe/test/tags?per_page=100&page=2')
+        .reply(200, [{name: 'v1.0.3'}])
+        .get('/repos/bcoe/test/tags?per_page=100&page=3')
+        .reply(200, [{name: 'v1.0.4'}])
+        .get('/repos/bcoe/test/tags?per_page=100&page=4')
+        .reply(200, [{name: 'v1.0.5'}])
+        .get('/repos/bcoe/test/tags?per_page=100&page=5')
+        .reply(200, [{name: 'v1.0.6'}]);
```

Comment:
Is this tied to an actual repo  you have set up, or is it mocked out?

Reply:
This is just mocked, I wanted to exercise the pagination logic.